### ISSUE TITLE
NEI gregtech material pages

### ISSUE THREAD

**Issue by MaGeBZz** [GTNewHorizons/GT-New-Horizons-Modpack#18191]
### Your GTNH Discord Username

magebzz_2

### Your Pack Version

2.7 Beta 4

### Your Server

Private Server

### Java Version

Java 17

### Type of Server

Don't know

### Your Expectation

Wanna look at stellite how you can do the dust for example and click on the "GregTech Material Parts"-Page

### The Reality

The "GregTech Material Parts"-Page is completely missing
![image](https://github.com/user-attachments/assets/63576605-6914-45d1-9676-e4425dc28b3c)

That's the case for:
- Stellite
- Quantum
- Proto-Halkonite Steel
- Abyssal Alloy
- Octiron
- Black Titanium
- Titansteel
- Botmium
- Cinobite A243
- Pikyonium 64B
- Arceus Alloy 2B
- Laurenium
- Lafium Compound
- HeLiCoPtEr
- Zeron-100
- Trinium Naquadah Carbonite
- HG-1223
- Selenium
- HS-188-A
- Arcanite
- Thallium
- Nitinol 60
Basically all alloys in ABS are missing it whatever why

### Your Proposal

Adding a stellite "GregTech Material Parts"-Page would be quite nice to fit better with all the other materials that have it

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by chochem** [comment]
I mean yea, they are not gregtech materials

**Comment by MaGeBZz** [comment]
Ok haven't noticed that because the fluids still say it's GT and not GT++ but yeah would be nice to get a material page anyways I would say but you already switched it to suggestion
